public/stylesheets/homepage/userbar.less

```diff
@@ -21,7 +20,7 @@ body {
 }
 
 .navbar-goggles {
-  background-color: @purple;
+  background-color: #682060;
```

Comment:
there was never a @purple, that was from me landing a change to that file too soon.